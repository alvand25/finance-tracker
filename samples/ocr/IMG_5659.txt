re) Jf oo * ae
a i A en ee eee
x x fos yt
a ioe
= option ; v ' .
a ae 5et
6 FA55 OY Rhea ty) AN sd
142 E. 14th 5t.
Nev york, NY 10003
5tare g0h4 - 212-529-4612
GhOH GAN (0 4 :00K4 DATLY
5ALE TRAM5AG FLON
Chity CHAL FEA LATTE Hix e4g9
Org FLOTE CORN CHIP5 $2.59
CREAM CHEE5E BRICK $3.98
2931.99
Items in Transaction 4
fidosic tee peas wht 8o
a derbaid b1) .9a
Baye AG PUCGHE 5e (RAR ACT HUN
: CU5TUNLA Cry
*Haatetiard ; pavattad tenga
f1ys CUR TALT LE5 Ay) Cotes 22 ae
Hi11e aeerat270yg Lin, Pera Led
Tak care Hace 5W
ky 5ardAu1ikee Yer Thiet bon
pau Pe1a d1 Pub jou feed,