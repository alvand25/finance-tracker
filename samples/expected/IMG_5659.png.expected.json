{
    "store": "Costco",
    "items": [
        {
            "description": "KIRKLAND BATTERIES",
            "price": 14.99
        },
        {
            "description": "ROTISSERIE CHICKEN",
            "price": 4.99
        },
        {
            "description": "KIRKLAND WATER 40PK",
            "price": 3.79
        }
    ],
    "subtotal": 23.77,
    "tax": 1.20,
    "total": 24.97,
    "payment_method": "VISA",
    "timestamp": "2024-04-15T10:45:00",
    "check_items_exactly": true
}